dev-util/intel-ocl-sdk/intel-ocl-sdk-18.1.0.015.ebuild

```diff
@@ -0,0 +1,60 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit rpm
+
+DESCRIPTION="Intel's implementation of the OpenCL standard"
+HOMEPAGE="https://www.intel.com/content/www/us/en/developer/articles/tool/opencl-drivers.html#cpu-section"
+SRC_URI="https://registrationcenter-download.intel.com/akdlm/irc_nas/vcp/15532/l_opencl_p_${PV}.tgz"
+
+LICENSE="Intel-SDP"
+SLOT="0"
+KEYWORDS="-* amd64"
+RESTRICT="bindist mirror"
+
+RDEPEND=">=virtual/opencl-3
+	sys-process/numactl"
+DEPEND=""
+PDEPEND="dev-libs/glib
+	media-libs/fontconfig
+	media-libs/freetype
+	x11-libs/libICE
+	x11-libs/libSM
+	x11-libs/libX11
+	x11-libs/libXext
+	x11-libs/libXrender
+	sys-libs/zlib"
+
+S="${WORKDIR}/l_opencl_p_${PV}"
+
+INTEL_CL="opt/intel/opencl_compilers_and_libraries_${PV}/linux"
+INTEL_INSTALL_PATH="/opt/intel/opencl-${PV}"
+INTEL_VENDOR_DIR=usr/lib/OpenCL/vendors/intel
+ALT_PV="${PV/.015/-015}"
+
+QA_PREBUILT="${INTEL_INSTALL_PATH}/*"
+
+src_unpack() {
+	default
+	rpm_unpack "${S}/rpm/intel-openclrt-${PV}-${ALT_PV}.x86_64.rpm"
```

Comment:
I'll look at this, rpm.eclass is pretty stale so it's possible this is right, not checked

Reply:
Issue is that `rpm_unpack` wants a relative path but that relative path is relative to what's in distdir. This RPM is located inside the downloaded tarball.